app/js/components/labOrderEntry/DraftOrder/index.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import DraftList from './DraftList';
+
+export class LabDraftOrder extends React.Component {
+  render() {
+    const { draftLabOrders, disableCancelButton, disableSaveButton } = this.props;
+    return (
+      <div className="draft-spacing draft-lab-layout">
+        <h5 className="h5-draft-header">
+          Unsaved Draft Orders ({draftLabOrders.length})
```

Comment:
would it be cool to still have this if `draftLabOrders.length` was zero ? Just asking though

Reply:
to indicate that there are no tests in the draft

Reply:
Done thanks